src/js/components/partner_leads/Index.js

```diff
@@ -1,7 +1,11 @@
 import React, {PropTypes} from 'react';
+import toastr from 'toastr';
 
 class PartnerLeads extends React.Component {
   render() {
+    toastr.options.positionClass = "toast-top-right";
+    toastr.options.preventDuplicates = true;
```

Comment:
Geez. Of course. I kind of knew something was weird about this implementation. Changed it. Thanks.